recipes/desserts/chocolate_muffins_with_liquid_core.adoc

```diff
@@ -34,7 +39,7 @@ Preparation:
 . Melt the bitter chocolate with the butter in a bain-marie and fold in the sugar-egg mass
 . Add flour and cacoa powder and stir it with the hand mixer.
 . Lube a muffin form or use paper forms, fill it with the mass
-. Bake it 15 minutes at 200°C (pre-heated), in the center
+. Bake it max. 15 minutes at 200°C/390F (pre-heated), in the center
```

Comment:
In the "normal" mode or with convection?

Reply:
specified